docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -163,11 +163,40 @@ The following command deletes the Alertmanager configuration in the Grafana Mimi
 mimirtool alertmanager delete
 ```
 
-#### Migrate Alertmanager configuration for UTF-8 in Alertmanager 0.27 and later
+#### Migrate Alertmanager configuration for UTF-8 in Mimir 2.12 and newer
 
-The migrate-utf8 command translates any matchers that are incompatible with the UTF-8 parser into
-equivalent matchers that are compatible by enforcing double quoting on the right hand side
-of the matcher. The translation does not change their behavior.
+This requires mimirtool version 2.12 or newer. You can check the version of mimirtool with
+`mimirtool version`.
+
+In accordance with [prometheus/prometheus#13095](https://github.com/prometheus/prometheus/issues/13095)
+and [prometheus/alertmanager#3486](https://github.com/prometheus/alertmanager/issues/3486), Mimir is
+adding support for UTF-8. To support UTF-8 in alerts, routes, silences and inhibition rules,
+Alertmanager has added a new parser for matchers that has a number of backwards incompatible changes.
+More information about these changes can be found [here](https://prometheus.io/docs/alerting/latest/configuration/#label-matchers).
+
+The migrate-utf8 command migrates an existing Alertmanager configuration in preparation for when
+UTF-8 is enabled in a Mimir installation. To do this, it translates matchers that are incompatible
+with UTF-8 into equivalent matchers that are compatible. This translation is backwards compatible
+and does not change the behavior of existing matchers, and works even in Mimir installations that
+do not have UTF-8 enabled.
+
+A lot of Alertmanager configurations will not need migrating. You can verify if an existing
+Alertmanager configuration needs to be migrated for UTF-8 by running the verify command:
```

Comment:
```suggestion
You don't need to migrate all Alertmanager configurations. To verify if you need to migrate an existing
Alertmanager configuration for UTF-8, run the verify command:
```

Reply:
> You don't need to migrate all Alertmanager configurations.

Some users might, it depends on what's in their configuration. We found that the number of affected users is very small (single digit %), but if a user has an incompatible line in all their configurations, then all will need migrating. If that makes sense?

Reply:
What about:

> Whether an Alertmanager configuration needs to be migrated depends on whether it contains matchers
incompatible with UTF-8. You can verify if an existing Alertmanager configuration needs to be
migrated for UTF-8 by running the verify command:

Reply:
I like this. Here's a small suggestion to avoid the double whether, as well as passive voice.

"The need to migrate an Alertmanager configuration depends on whether it contains matchers
incompatible with UTF-8. To determine whether you need to migrate an existing configuration, run the verify command:"